File: problems/2833-furthest-point-from-origin/solution.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int left = 0;
        int right = 0;
        int underscore = 0;
        for (char c : moves.toCharArray()) {
            if (c == 'L') {
                left++;
            } else if (c == 'R') {
                right++;
            } else {
                underscore++;
            }
        }
        return Math.abs(right - left) + underscore;
    }
}

File: problems/2833-furthest-point-from-origin/solution.c
int furthestDistanceFromOrigin(char* moves) {
    int right = 0;
    int left = 0;
    int underscore = 0;
    int i = 0;
    while (moves[i] != '\0') {
        if (moves[i] == 'R') {
            right++;
        } else if (moves[i] == 'L') {
            left++;
        } else {
            underscore++;
        }
        i++;
    }
    if (right > left + underscore) {
        return right - left - underscore;
    } else {
        return underscore - (right - left);
    }
}